wsaa.py

```diff
@@ -54,6 +54,7 @@
     from cryptography.hazmat.primitives import serialization
     from cryptography.hazmat.primitives.asymmetric import rsa
     from cryptography.hazmat.bindings.openssl.binding import Binding
+    from cryptography.hamzat.primitives.serialization import pkcs7
```

Comment:
Hi @HanslettTheDev! here is the typo flagged in [discord](https://discord.com/channels/1109520980453371946/1109527189151551518/1111788247408975892)

Reply:
yes, let me check the docs and see if the primitives section doesn't use pkcs7

Reply:
Oh I see the error now
I wrote instead **hamzat** instead of **hazmat**. Thanks Nico!